specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/securitydomain.json

```diff
@@ -0,0 +1,336 @@
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "name": "vaultBaseUrl",
+        "description": "The vault name, for example https://myvault.vault.azure.net.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/securitydomain/download": {
+      "post": {
+        "tags": [
+          "HSMSecurityDomain"
+        ],
+        "operationId": "HSMSecurityDomainDownload",
+        "description": "Retrieves Security domain from HSM enclave",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "Certificates",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CertificateSet"
+            },
+            "description": "Security domain download operation requires customer to provide three certificates containing public key in JWK format."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Security domain download response",
+            "schema": {
+              "$ref": "#/definitions/SecurityDomainObject"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Retrieve Security domain": {
+            "$ref": "./examples/securitydomaindownloadpost-example.json"
+          }
+        }
+      }
+    },
+    "/securitydomain/transferkey": {
+      "get": {
+        "tags": [
+          "HSMSecurityDomain"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "operationId": "HSMSecurityDomainTransferKey",
+        "description": "Retrieve security domain transfer key",
+        "responses": {
+          "200": {
+            "description": "Security domain transfer key operation",
+            "schema": {
+              "$ref": "#/definitions/TransferKey"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Retrieve security domain transfer key": {
+            "$ref": "./examples/securitydomaintransferkey-example.json"
+          }
+        }
+      }
+    },
+    "/securitydomain/upload": {
+      "post": {
+        "tags": [
+          "HSMSecurityDomain"
+        ],
+        "operationId": "HSMSecurityDomainUpload",
```

Comment:
This operation defines a Azure-AsyncOperation header, do we want to use the autorest extension x-ms-long-running to get LRO generation?